### PR TITLE
Added spanish language

### PR DESCRIPTION
**Pull request by scodx**
Added spanish language as es_MX

### CODE REVIEW

**Comment by cuonic** [comment]
Thanks for the contribution ! Could you try fixing the error generated during the language unit tests ?

**Comment by scodx** [comment]
Yeah I don´t know why the tests are failing, I added a new line at the end and still getting failed tests.

File: languages/es_MX.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+$lang['email_reset_subject'] = '%s - Petición para reinicia contraseña';
+$lang['email_reset_body'] = 'Hola,<br/><br/>Para reiniciar tu contraseña por favor da click en el siguiente enlace:<br/><br/><strong><a href="%1$s/%2$s">%1$s/%2$s</a></strong><br/><br/>Después, necesitas usar la siguiente clave de reinicio: <strong>%3$s</strong><br/><br/>Si no solicitaste reiniciar tu contraseña en %1$s por favor ignora este mensaje.';
+$lang['email_reset_altbody'] = 'Hola, ' . "\n\n" . 'Para reiniciar tu contraseña por favor da click en el siguiente enlace:' . "\n" . '%1$s/%2$s' . "\n\n" . 'Después, necesitas usar la siguiente clave de reinicio: %3$s' . "\n\n" . 'Si no solicitaste reiniciar tu contraseña en %1$s por favor ignora este mensaje.';
+
+$lang['account_deleted'] = "La cuenta se ha borrado con éxito.";
+$lang['function_desabled'] = "Esta función ha sido desactivada.";
```
Comment (cuonic) [inline languages/es_MX.php]: I think this is your problem, take a look at https://github.com/PHPAuth/PHPAuth/blob/master/languages/en_GB.php#L71

It's "disabled" not "desabled"

**Comment by scodx** [comment]
yay!

**Comment by cuonic** [comment]
Great, thanks a lot !

**Pull request merged**